core/conf/demo/server_config.yaml

```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
 #----------------------+------------------------------------------------------------+------------+-----------------+
 tracing_config:
   json_config_path:
+
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# WAL Config           | Description                                                | Type       | Default         |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# enable               | Switch of function WAL.                                    | Boolean    | false           |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# recovery_error_ignore| Whether ignore the error which happens during WAL recovery | Boolean    | true            |
```

Comment:
Whether to ignore logs with errors that happens during WAL recovery. If true, when Milvus restarts for recovery and there are errors in WAL log files, log files with errors are ignored. If false, Milvus does not restart when there are errors in WAL log files.

Reply:
Please apply this change to other files.